### ISSUE TITLE
Disable HTML in Mail

### ISSUE THREAD

**Issue by hensschenit** [nextcloud/mail#1571]
Hey all together,

would there be a chance that you implement a function/parameter to disable HTML in E-Mails?

Or is that already one?

Regards Tommy

**Comment by ChristophWurst** [comment]
Could you elaborate? Are you talking about incoming or outgoing messages?

Btw, the template chooser and the templates are there for a reason. https://github.com/nextcloud/mail/issues/new/choose Use them :wink:

**Comment by hensschenit** [comment]
Oh sorry! 
I had chosen "Feature Request", but removed the inline content.

I mean incoming messages.

**Comment by ChristophWurst** [comment]
Okay, so what do you mean with *disable HTML* then? Extract text from the HTML? Or show the text-only if it's a multipart?

**Comment by hensschenit** [comment]
So if it in HTML Format only --> extract only text
And if the mail comes per multipart --> show only the text

**Comment by hensschenit** [comment]
So if it in HTML Format only --> extract only text
And if the mail comes per multipart --> show only the text

**Comment by ChristophWurst** [comment]
Is there any other client that does this? I'm pretty sure most emails will look pretty unreadable.

**Comment by hensschenit** [comment]
Currently i use Open-Xchange and there is an option to deactivate HTML in mails.
Also Outlook has an option for that.

And in Open-Xchange it is almost readable.

**Comment by dcrobertson01** [comment]
Thnunderbird has the option to display the message body as Original HTML, Simple HTML or Plain Text.

I'd like to see this too - don't understand why people need to use HTML for simple email messages ... though I suppose it is better than sending stuff as word documents with 3mb logos ...

**Comment by yoyko** [comment]
I would also like an option to prefer showing the `text/plain` part if a message contains it. I find it much easier to read those: just the text I'm actually interested in and no distractions.

I have mutt set up that way and I actually had some people ask me whether I have some fancy filter set up that gets rid of the junk ;-) However it's true that sometimes I had to open the html to be able to properly read the message, but those cases are pretty rare.

> So if it in HTML Format only --> extract only text

I guess the current sanitized display is OK for those...
Btw, I just did a search on my inbox for the last month: there was about 15% of messages without a `text/plain` part, none of them were from a real person ;-) (though not everything was "junk")
